README.md

```diff
@@ -1,75 +1,289 @@
+    - [Prerequisites](#prerequisites)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors <a name="authors"></a>](#-authors-)
+  - [🔭 Future Features <a name="future-features"></a>](#-future-features-)
+  - [🤝 Contributing <a name="contributing"></a>](#-contributing-)
+  - [⭐️ Show your support <a name="support"></a>](#️-show-your-support-)
+  - [🙏 Acknowledgments <a name="acknowledgements"></a>](#-acknowledgments-)
+  - [❓ FAQ <a name="faq"></a>](#-faq-)
+  - [📝 License <a name="license"></a>](#-license-)
 
+<!-- PROJECT DESCRIPTION -->
 
-## Getting Started
+# 📖 [your_project_name] <a name="about-project"></a>
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+> The `Doctor-Appointment Project` project is a appointment booking project split to API and Frontend app.
 
+**[DOCTOR-APPOINTMENT PROJECT ]** is designed to allow users to book Dcoctor Appoinment. Our API has predictable resource-oriented URLs, accepts form-encoded request bodies, returns JSON-encoded responses, and uses standard HTTP response codes, authentication, and verbs. A user friendly frontend app enables uesrs to easily manage their appointments and choose a Doctor from the list.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- ROR
+- React
+- Rspec
+- Git
+- Github
+- Rubocop
+- RSpec
+- Stylelint
+
+### Tech Stack <a name="tech-stack"></a>
+
+> API built with ROR and provides a number of endpoints to enable developers to utilize API. Database id designed with postgreSQL and ensures all associated data is being written or fetched. React frontend app is a userfriendly UI to demonstrate integration and API utility
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Authentication]** Users are required to sign up to app to get access to all features of the application 
+- **[Authorization]** Users are authourized accordingly to roles assigned with a help of CANCANCAN 
+- **[Documented]** API is documented and esures easy integration with a custom frontend app built with tech you prefer to use
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```

Comment:
- [ ]  If you are not using a live demo for the backend, kindly remove it OR you could indicate this is a placeholder for a future deployment by writing something like "Coming soon" or "TBA".  Just don't leave the broken link.  💪